activestorage/test/models/attached/one_test.rb

```diff
@@ -16,6 +16,22 @@ class ActiveStorage::OneAttachedTest < ActiveSupport::TestCase
     ActiveStorage::Blob.all.each(&:delete)
   end
 
+  test "creating a record with a File as attachable attribute" do
+    @user = User.create!(name: "Dorian", avatar: File.open("test/fixtures/files/dorian.jpg"))
```

Comment:
Since files are defined in `test/fixtures/files`, would a call to `file_fixture(...).open` return the same `File` value as `File.open` with the full path?

Reply:
yeah that seems to do the same thing:

```
>> File.open("test/fixtures/files/dorian.jpg")
=> #<File:test/fixtures/files/dorian.jpg>
>> file_fixture("dorian.jpg").open
=> #<File:/Users/dorianmariefr/src/rails/activestorage/test/fixtures/files/dorian.jpg>
```